src/service/xhr-impl.js

```diff
@@ -124,6 +125,11 @@ export class Xhr {
     dev().assert(
         creds === undefined || creds == 'include' || creds == 'omit',
         'Only credentials=include|omit support: %s', creds);
+
+    if (init.body instanceof FormDataWrapper) {
```

Comment:
Please add a comment explaining why we're doing this.

Reply:
Done.